fh_immuta_utils/client.py

```diff
@@ -535,6 +548,20 @@ def tag_data_source(self, id: int, tag_data: List[Dict[str, Any]]) -> bool:
         res.raise_for_status()
         return True
 
+    def update_data_source_tags_in_bulk(
+        self, ids: List[int], tag_data: List[Dict[str, Any]]
+    ) -> bool:
+        """
+        Adds tags to a list of data sources.
+        :param ids: data source ids
+        :param tag_data: list of tag dicts to apply to the data sources
+        :return: True
+        """
+        self.put(
+            "dataSource/bulk/tags", data={"ids": ids, "update": {"tags": tag_data}}
```

Comment:
i.e. 1:1 correspondence between list of ids and list of tag_data?

Reply:
nvm, from the usage it appears apply all tags to all ids in the list

Reply:
yep - added to the docstring to clarify this.